src/PassengerClass.js

```diff
@@ -0,0 +1,16 @@
+const { bookshelf, Passenger, Passengers, knex } = require('../database/connection.js')
+const commands = require('./commands.js')
+
+class PassengerClass {
```

Comment:
I see this naming convention throughout your project where a class is named using the word `Class` - I would suggest using an identifier that suggests the domain object itself, and omit the word `Class`.  `Passenger` by itself, for example, adequately identifies the purpose of this class.  `Class` is redundant.